apps/src/templates/teacherNavigation/lessonMaterials/LessonMaterialsContainer.tsx

```diff
@@ -30,25 +38,20 @@ interface LessonMaterialsData {
   versionYear?: number;
 }
 
-const lessonMaterialsCachedLoader = _.memoize(async (unitId: number) =>
+const lessonMaterialsApiCall = (unitId: number) =>
```

Comment:
Move the memoize into the class so it is not a static memo and instead based on the class. This was causing lots of weird test failures.